docs/api-reference/results/get-results-pdf.mdx

```diff
@@ -5,13 +5,35 @@ openapi: "GET /v3/order/{order_id}/result/pdf"
 
 <RequestExample>
 
+```bash cURL
+curl --request GET \
+     --url '{{BASE_URL}}/v3/order/<order_id>/result/pdf' \
+     --header 'accept: application/json' \
+     --header 'x-vital-api-key: {YOUR_KEY}'
+```
+
 ```python Python
 from vital import Client
 
-client = Client(api_key, "sandbox")
+client = Client(
+  api_key=<API_KEY>,
+  environment="sandbox",
+  region="us"
+)
 
 data = client.LabTests.get_results_pdf("<order_id">)
+```
+
+```javascript Node
+const { VitalClient } = require("Vital");
```

Comment:
Is it "Vital"? I thought it was vital-node?

Reply:
Good question. I copied from other places in the docs, e.g. https://docs.tryvital.io/api-reference/user/create-user.

Can double check.

Reply:
Didn't double check yet but I will merge this in the meantime. If it turns out to be wrong we'd have to change it everywhere anyway so can do that in a separate PR.